editor/plugins/org.fusesource.ide.branding/src/org/fusesource/ide/branding/wizards/NewCamelTestWizard.java

```diff
@@ -101,20 +98,15 @@ public boolean performFinish() {
 			@Override
 			public void run(IProgressMonitor monitor) throws InvocationTargetException,
 			InterruptedException {
-				if (monitor == null) {
-					monitor = new NullProgressMonitor();
-				}
-				monitor.beginTask(WizardMessages.NewTestCaseCreationWizard_create_progress, 2);
+				SubMonitor subMonitor = SubMonitor.convert(monitor, WizardMessages.NewTestCaseCreationWizard_create_progress, 2);
 				
 				fPage1.superClassChanged();
-				monitor.worked(1);
+				subMonitor.setWorkRemaining(1);
 				try {
-					fPage1.createType(monitor);
+					fPage1.createType(subMonitor.split(1));
 					monitor.worked(1);
```

Comment:
I think there should be
```
subMonitor.worked(1);
```

Reply:
good catch, in fact the line can even be removed